### DISCUSSION TITLE
Is the dynamic federation react support hmr refresh through devRemote ?

### DISCUSSION THREAD

**Discussion by Carrgan** [nrwl/nx#27803] [Questions]
I have been generated the example by `nx g @nx/react:host --name=host --directory=apps --remotes=remote1 --dynamic=true`

Then I use `nx run host:serve:development --devRemotes=remote1` to start.

When I change remote1 and save, I can see, the running console is building and the browser console have hmr log printed. But the web page not refresh. 

And if I change `loadRemoteModule` back to `import()` in remote1 then everything is working fine.

loadRemoteModule is import form `import { loadRemoteModule } from "@nx/react/mf";`

Is the dynamic federation react is not support hmr ? If I want to development the remote, how can i do ?

**Comment by LeonieNFarmer** [comment]
I have a similar issue. I was able to get hmr working when using nx and dynamic module federation with webpack. Once I changed the bundler to rsPack the hmr only works for the shell app.
